workflows/misc/cleanup_production.go

```diff
@@ -0,0 +1,153 @@
+package miscworkflows
+
+import (
+	"context"
+	"fmt"
+	"github.com/bcc-code/bcc-media-flows/activities"
+	"github.com/bcc-code/bcc-media-flows/activities/cantemo"
+	avidispine "github.com/bcc-code/bcc-media-flows/activities/vidispine"
+	cantemoservice "github.com/bcc-code/bcc-media-flows/services/cantemo"
+	wfutils "github.com/bcc-code/bcc-media-flows/utils/workflows"
+	"go.temporal.io/sdk/workflow"
+	"path"
+	"regexp"
+	"strings"
+	"time"
+)
+
+var regIllegalChars = regexp.MustCompile(`([^a-zA-Z0-9\-\._]|\s)`)
+
+type RenameCantemoFileSpecialParams struct {
+	File    cantemoservice.Objects
+	OldName string
+	OldPath string
+	Storage string
+}
+
+func RenameCantemoFileSpecial(ctx context.Context, params RenameCantemoFileSpecialParams) (any, error) {
```

Comment:
can you document what this does diffrently from RenameFile?
maybe it can have a better name